system/jlib/jcomp.cpp

```diff
@@ -814,7 +814,7 @@ void CppCompiler::setTargetBitLength(unsigned bitlength)
     case GccCppCompiler:
         switch (bitlength)
         {
-        case 32: option = "-m32"; break;
```

Comment:
This flag may still be required on intel versions - should make this change ARM only I think